purchases/src/main/kotlin/com/revenuecat/purchases/SubscriptionInfo.kt

```diff
@@ -74,6 +76,22 @@ class SubscriptionInfo(
      * The transaction id in the store of the subscription.
      */
     val storeTransactionId: String?,
+    /**
+     * Date when the subscription will auto-resume.
+     */
+    val autoResumeDate: Date?,
+    /**
+     * The display name of the subscription.
+     */
+    val displayName: String?,
+    /**
+     * Paid price for the subscription.
+     */
+    val price: Price?,
+    /**
+     * The identifier of the product plan.
+     */
+    val productPlanIdentifier: String?,
```

Comment:
To avoid the breaking change, we should add a constructor without all these fields defaulting to null and deprecate it. Alternatively, we could default all of them to `null` and add a `@JvmOverloads`, though I think that would still break for non-named arguments since we have a non-optional parameter after this (`requestDate`, though we could always move all these to after that one I guess)